shared/js/utils.js

```diff
@@ -36,6 +36,12 @@ require.scopes.utils = ( () => {
         }
     }
 
+    function capitalizeFirstLetter (string) {
+        if (typeof string !== 'string') { return '' }
+
+        return string[0].toUpperCase() + string.substring(1)
+    }
```

Comment:
In the Site template, I am using an npm package called `titleize` for this. Please use that instead on the front end. This utils file is for the background process.